packages/styles/scss/components/card-link/index.scss

```diff
@@ -9,6 +9,7 @@
 @import '../../globals/vars';
 @import '../../globals/mixins.scss';
 
+@import '~carbon-components/scss/components/link/link';
```

Comment:
@kennylam we can't import with `~`, as that's a webpack specific thing

Reply:
@kennylam not sure if it's something on my end, this was marked as resolved but I don't see a change applied.